src/app/components/audios/audio-view/audio-view.component.spec.ts

```diff
@@ -167,6 +168,7 @@ describe('AudioViewComponent', () => {
           { provide: CaseService, useValue: fakeCaseService },
           { provide: ErrorMessageService, useValue: fakeErrorMessageService },
           { provide: AppConfigService, useValue: appConfigServiceMock },
+          DatePipe,
```

Comment:
Without the utc setting in here, would BST dates fail time checks?

Reply:
I'll look into it

Reply:
Yes they would but I'm not asserting on anytimes. Added it anyway for completeness